### DISCUSSION TITLE
css for adding bar at the bottom of slides

### DISCUSSION THREAD

**Discussion by SoumayaMauthoorMOJ** [marp-team/marp#521] [Q&A] [answered]
I'm creating a [custom theme](https://github.com/ministryofjustice/marp-moj-theme). Is there a way to add a bar or box at the bottom of slides without passing in an image? Similar to [this theme](https://github.com/kaisugi/marp-theme-academic/blob/main/demo.002.png) , but at the bottom instead of at the top?

**Comment by yhatt** [comment, marked as answer]
```css
section {
  box-shadow: inset 0 -10px 0 red;
}
```

[Show the example at JSFiddle...](https://jsfiddle.net/bz6nax9w/)

**Reply by SoumayaMauthoorMOJ** [reply]
Thank you!
